### PR TITLE
Creation tx details

### LINKED ISSUE #848: Details screen for creation transaction
- Data is taken from the list item
- NO pull to refresh
- "Implementation/Master copy" and "Factory" address may be not available

### PR DESCRIPTION
**Pull request by biafra23**
Closes #848 .

Changes proposed in this pull request:

Screenshot_20200909-113831

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -201,4 +201,7 @@
     <string name="channel_tx_notifications_name">Tx Notifications</string>
     <string name="channel_tx_notifications_description">Notifications about incoming transfers and processed transactions.</string>
     <string name="tx_list_creation">Safe created</string>
+    <string name="tx_details_creation_creator_address">Creator address</string>
```
Comment (elgatovital) [inline app/src/main/res/values/strings.xml:204]: better to group all tx_details strings together (see line 166)

File: app/src/main/res/navigation/main_nav.xml [hunk 2]
```diff
@@ -94,14 +104,64 @@
...
 
+    </fragment>
+
+    <fragment
+        android:id="@+id/creationTransactionDetailsFragment"
+        android:name="io.gnosis.safe.ui.transactions.details.CreationTransactionDetailsFragment"
+        android:label="CreationTransactionDetailsFragment"
+        tools:layout="@layout/fragment_transaction_details_creation">
+
         <action
```
Comment (elgatovital) [inline app/src/main/res/navigation/main_nav.xml:124]: why is this action inside tx details fragment?

File: app/src/main/res/layout/fragment_transaction_details_creation.xml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/default_margin"
+                android:text="@string/tx_details_advanced" />
+
+        </LinearLayout>
+
+    </com.google.android.material.appbar.AppBarLayout>
+    <!--
+                    statusText = displayString(transaction.status),
```
Comment (elgatovital) [inline app/src/main/res/layout/fragment_transaction_details_creation.xml:46]: do wee need these comments?

File: app/src/main/res/navigation/main_nav.xml (hunk 2 above)
Comment (biafra23) [reply app/src/main/res/navigation/main_nav.xml]: Removed in 4a948ac8ff2a0696ec41e88674941c788f6cb3eb

File: app/src/main/res/layout/fragment_transaction_details_creation.xml (hunk 3 above)
Comment (biafra23) [reply app/src/main/res/layout/fragment_transaction_details_creation.xml]: No.

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/CreationTransactionDetailsFragment.kt [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+import pm.gnosis.svalinn.common.utils.visible
+import pm.gnosis.utils.asEthereumAddress
+
+class CreationTransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDetailsCreationBinding>() {
+
+    override fun screenId() = ScreenId.TRANSACTIONS_DETAILS
+
+    private val navArgs by navArgs<CreationTransactionDetailsFragmentArgs>()
+
+    private val statusTextRes by lazy { navArgs.statusTextRes }
```
Comment (elgatovital) [inline app/src/main/java/io/gnosis/safe/ui/transactions/details/CreationTransactionDetailsFragment.kt:27]: can we just pass status and reuse getColorForStatus that we currently have in TxDetailsFragment (we should then move it to TxUtils)

**Review by elgatovital — APPROVED** [review]

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/CreationTransactionDetailsFragment.kt (hunk 4 above)
Comment (biafra23) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/CreationTransactionDetailsFragment.kt:27]: I just saw this after I merged.

Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/CreationTransactionDetailsFragment.kt:27]: n.p. You could add it to the next pr if this makes sense.

**Pull request merged**